grocy/rootfs/patches/fix_braindamage.patch

```diff
@@ -7,8 +7,13 @@ index fceaaa0..bfc76e6 100644
  		Grocy.Mode = '{{ GROCY_MODE }}';
 -		Grocy.BaseUrl = '{{ $U('/') }}';
 -		Grocy.CurrentUrlRelative = "/" + window.location.href.split('?')[0].replace(Grocy.BaseUrl, "");
-+		Grocy.BaseUrl = window.location.origin + '{{ $U('/') }}';
-+		Grocy.CurrentUrlRelative = "/" + window.location.pathname.replace(Grocy.BaseUrl, "");
++		if('{{ $U('/') }}'.includes('http')) {
++			Grocy.BaseUrl = '{{ $U('/') }}';
++			Grocy.CurrentUrlRelative = "/" + window.location.href.split('?')[0].replace(Grocy.BaseUrl, "");
```

Comment:
`window.location.href.split('?')`

Whatever the solution, that piece of code is dead wrong and should not be part of it.